src/interfaces.py

```diff
@@ -11,6 +11,20 @@
 T = TypeVar("T")
 
 
+class JobProtocol(Protocol):
+    """Represent a Job class from src/job.py."""
+
+    name: str
+    source: Any
+    destination: Any
```

Comment:
```suggestion
    source: Source
    destination: Destination
```

Reply:
Actually, we don't even need the job interface to have these fields. It only needs a run method.